src/main/target/NUCLEOH7A3ZI/config.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "platform.h"
+
+#ifdef USE_TARGET_CONFIG
+
+#include "config_helper.h"
+
+#include "io/serial.h"
+
+#include "pg/sdcard.h"
+
+static targetSerialPortFunction_t targetSerialPortFunction[] = {
+    { SERIAL_PORT_USART1, FUNCTION_MSP },
+    { SERIAL_PORT_USART3, FUNCTION_MSP },
```

Comment:
What is the reason for these - if devices producing serial input (RX, GPS, ...) are connected to these UARTs predefining them as MSP will result in hard lockups during boot, so not sure we want these as defaults.

Reply:
These were originally set up for UART device testing.
I think USART3 over PD8 and PD9 can be left this way as it is connected by default to VCP through parasitic ST-LINK on large Nucleos (Nucleo-144) including the Nucleo-H7A3 which this target is made for.

USART1 may be removed.

Thoughts?

P.S.
Reminder: NUCLEOH743 target should be fixed for the same reason.

Reply:
Sounds right to me - if it is connected to USB by default then keeping MSP on it makes sense - will be nice to not have to use the secondary (VCP) USB connector on these boards.